### PR TITLE
Adding docs for approving Airtable entries.

### PR DESCRIPTION
**Pull request by Quinncuatro**
Took me longer than I'd like to admit to find the `Approval` column in Airtable. Added some verbage to the README to let others know they need to approve entries then manually trigger a Netlify deployment to have changes show up.

### CODE REVIEW

**Comment by boborchard** [comment]
Hey @Quinncuatro, thank you for the PR! IMHO since the change is to the starter Readme.md we probably don’t need to do a version bump. Other than that it looks good to me!

_p.s. thanks for adding the package lock to gitignore!_

File: examples/starter/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "gatsby-starter-service-relief",
   "private": true,
-  "version": "1.0.8",
+  "version": "1.0.9",
```
Comment (DSchau) [inline examples/starter/package.json]:
```suggestion
  "version": "1.0.8",
```

File: examples/starter/package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
   },
   "dependencies": {
     "gatsby": "^2.19.47",
-    "gatsby-theme-service-relief": "^0.0.3",
+    "gatsby-theme-service-relief": "^0.0.4",
```
Comment (DSchau) [inline examples/starter/package.json]:
```suggestion
    "gatsby-theme-service-relief": "^0.0.3",
```

File: themes/gatsby-theme-service-relief/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "gatsby-theme-service-relief",
-  "version": "0.0.3",
+  "version": "0.0.4",
```
Comment (DSchau) [inline themes/gatsby-theme-service-relief/package.json]:
```suggestion
  "version": "0.0.3",
```

**Review by DSchau — CHANGES_REQUESTED** [review]
Versioning is handled automatically by Lerna, so we'll want to reset those versions. Otherwise, this looks great; thank you!

**Comment by boborchard** [comment]
Hey @Quinncuatro, there were a few changes requested by Dustin and it looks like there were some extra commits added as well. We'll get your work merged if you can clean it up so it only has the changes for the `Readme.md`

Thanks!

**Comment by Quinncuatro** [comment]
Yeah, my bad. Tried spinning up my own version of the theme as a monorepo but haven't used monorepos before. Split off to a single repo for the theme. Let me clean this up.

**Comment by Quinncuatro** [comment]
Sorry about all that. Should be a clean merge, now. @boborchard

**Comment by Quinncuatro** [comment]
Screw it, I don't like the messy history. Hold on.

**Pull request closed without merging**